assets/js/modules/reader-revenue-manager/components/dashboard/PublicationApprovedOverlayNotification.js

```diff
@@ -73,12 +88,16 @@ export default function PublicationApprovedOverlayNotification() {
 	 * - The notification UI is enabled.
 	 * - The user is not in view-only mode.
 	 * - The user is on the main dashboard.
+	 * - The publication onboarding state has changed.
+	 * - The publication onboarding state is complete.
 	 */
 	const shouldShowNotification =
 		isDismissed === false &&
 		showApprovedNotificationUI === true &&
 		! isViewOnly &&
-		dashboardType === VIEW_CONTEXT_MAIN_DASHBOARD;
+		dashboardType === VIEW_CONTEXT_MAIN_DASHBOARD &&
+		initialPublicationOnboardingStateChanged.current === true &&
+		publicationOnboardingState === ONBOARDING_COMPLETE;
```

Comment:
Currently, the overlay notification no longer appears for me, because no one sets the value for `UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION` in the `CORE_UI` store, which is only done when the sync is run on the client-side.

The logic here should be changed so that the notification is displayed when either the UI store value is set, OR, the new conditions are met.

Somewhat like the following:

```js
const shouldShowNotification =
	isDismissed === false &&
	! isViewOnly &&
	dashboardType === VIEW_CONTEXT_MAIN_DASHBOARD &&
	( showApprovedNotificationUI === true ||
		( initialPublicationOnboardingStateChanged.current === true &&
			publicationOnboardingState === ONBOARDING_COMPLETE ) );
```

Could you please investigate and ensure that the notification appears?